### PR TITLE
RELATED: TMA-332 Support transfer tag objects in Release brick

### PR DESCRIPTION
**Pull request by vhtien**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/482/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/420/ : SUCCESS in 42m 56s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/156/ : FAILURE in 8s

File: lib/gooddata/lcm/actions/synchronize_tag_objects.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    class SynchronizeTagObjects < BaseAction
```
Comment (panjan) [inline lib/gooddata/lcm/actions/synchronize_tag_objects.rb]: Can we add a short description to the class?

File: lib/gooddata/lcm/actions/synchronize_tag_objects.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        description 'Synchronization Info'
+        param :synchronize, array_of(instance_of(Type::SynchronizationInfoType)), required: true, generated: true
+
+        description 'Tag Name'
+        param :production_tag, instance_of(Type::StringType), required: false
+      end
+
+      class << self
+        def call(params)
+          # Check if all required parameters were passed
```
Comment (panjan) [inline lib/gooddata/lcm/actions/synchronize_tag_objects.rb]: minor: pointless comment

File: lib/gooddata/lcm/actions/synchronize_tag_objects.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+              results << {
+                from: from_project,
+                to: pid,
+                tag: params.production_tag,
+                status: 'ok'
+              }
+            end
+          end
+
+          # Return results
```
Comment (panjan) [inline lib/gooddata/lcm/actions/synchronize_tag_objects.rb]: minor: pointless comment

File: lib/gooddata/lcm/actions/synchronize_tag_objects.rb (hunk 1 above)
Comment (vhtien) [inline lib/gooddata/lcm/actions/synchronize_tag_objects.rb]: fixed it

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/485/ : FAILURE in 16s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/423/ : FAILURE in 42m 30s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/161/ : SUCCESS in 2m 09s

**Pull request merged**